src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1049,6 +1049,31 @@ public String getQuestionText(String feedbackQuestionId) {
 
         return sortedMap;
     }
+    
+    public Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> getQuestionResponseMapSortedByRecipient() {
+        if (questions == null || responses == null) {
+            return null;
+        }
+
+        Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> sortedMap =
+                new TreeMap<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>();
+
+        for (FeedbackQuestionAttributes question : questions.values()) {
+            sortedMap.put(question, new ArrayList<FeedbackResponseAttributes>());
+        }
+
+        for (FeedbackResponseAttributes response : responses) {
+            List<FeedbackResponseAttributes> responsesForQuestion = sortedMap
+                    .get(questions.get(response.feedbackQuestionId));
```

Comment:
Perhaps you could extract `response.feedbackQuestionId` into a local variable to make more readable